index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/style.css" />
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link href="http://fonts.cdnfonts.com/css/lato" rel="stylesheet" />
+    <title>Document</title>
+  </head>
+  <body>
```

Comment:
- [ ] **Required:** Kindly observe that the `Home` page has a uniform margin.

Your Design | Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/163442910-00eec970-cf32-4238-adc0-6c00e3787db7.png) | ![image](https://user-images.githubusercontent.com/27709832/163442534-60b3f10a-d40c-4d21-be44-65b265b510c1.png)

- [ ] **Required:** The page title/heading has a background image and the size is bigger in contrast to the current design you have.

Your Design | Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/163443531-351aea37-0e75-4e62-af2c-7219a6bac105.png) | ![image](https://user-images.githubusercontent.com/27709832/163443585-0bd7484b-8cff-406a-8f97-1da587a2b573.png)
- The title `font-family` is cocogoose 
- The `size` and `font-family`/`font-weight` of this section is bigger.

Your Design | Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/163443945-5655130d-142b-46f7-b286-e40cf1b80b98.png) | ![image](https://user-images.githubusercontent.com/27709832/163444177-3e54e806-28e3-4759-bea8-c9e0343a20f2.png)